### ISSUE TITLE
Crash on Startup

### ISSUE THREAD

**Issue by rupefelo** [brenner-tobias/addon-cloudflared#168]
### The problem

After starting up and configuring the tunnel through cloudflare successfuly, cloudflared crashes. 

### What version of Cloudflared has the issue?

2.0.13

### What type of installation are you running?

Home Assistant OS

### Add-on YAML Configuration

```yaml
additional_hosts: []
external_hostname: ********.ml
tunnel_name: homeassistant
tunnel_token: ""
```

### Anything in the logs that might be useful for us?

```txt
2022-08-27T17:00:17Z INF Starting tunnel tunnelID=991a4f1e-967a-49a5-980e-5cdac4f04f1b
2022-08-27T17:00:17Z INF Version 2022.8.2
2022-08-27T17:00:17Z INF GOOS: linux, GOVersion: go1.18.5, GoArch: amd64
2022-08-27T17:00:17Z INF Settings: map[config:/tmp/config.json cred-file:/data/tunnel.json credentials-file:/data/tunnel.json loglevel:info metrics:0.0.0.0:36500 no-autoupdate:true origincert:/data/cert.pem]
2022-08-27T17:00:17Z INF Generated Connector ID: d4b42fb0-1b24-4bed-8a53-4414360bc096
2022-08-27T17:00:17Z INF Initial protocol quic
2022-08-27T17:00:17Z INF Starting metrics server on [::]:36500/metrics
2022-08-27T17:00:17Z INF You requested 4 HA connections but I can give you at most 0.
2022-08-27T17:00:17Z INF Tunnel server stopped
2022-08-27T17:00:17Z ERR Initiating shutdown error="there are no free edge addresses left to resolve to"
2022-08-27T17:00:18Z INF Metrics server stopped
there are no free edge addresses left to resolve to
[18:00:18] WARNING: cloudflared crashed, halting add-on
```

### Steps to reproduce the issue

Just starting the addon

**Comment by omershtricker** [comment]
I have the exact same issue, it is working just fine for the first time but if for some reasons i stop the plugin(either because of homeassistant restart or change settings ) i have this issue, i have no idea why, i tried couple of things:
A. I tried to uninstall and install again - didn’t work 
B. I tried to the delete all files option(didn’t work )
C. I tried resotre home assistant to backup from before i did it (no luck either)
D. I tried to move my domain to a new cloudflare account(nope)
E. Use new domain name. (I tried both a .com and a free .tk nothing work)
F. Use a vpn. Since maybe cloudflare band my ip or something(not working)

the only thing that fix that(until the next restart ) is to just give it 24 hours and try again usually after 24 hours it working again
